src-backend/registration/tests/test_login.py

```diff
@@ -0,0 +1,41 @@
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from rest_framework.authtoken.models import Token
+from nose.tools import assert_equals, assert_true, assert_false
+import json
+
+
+class LoginTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_invalid_method_cannot_login(self):
+        response = self.client.get('/auth/login')
+        assert_equals(response.status_code, 405)  # Method not allowed
```

Comment:
We could use http://www.django-rest-framework.org/api-guide/status-codes/ which basically will be `status.HTTP_405_METHOD_NOT_ALLOWED` instead of adding it in a comment